e2e/test/scenarios/native-filters/helpers/e2e-date-filter-helpers.js

```diff
@@ -77,5 +75,5 @@ export function setAdHocFilter({
     cy.findByText(/^Include/).click();
   }
 
-  cy.button("Add filter").click();
+  cy.button(buttonLabel).click();
```

Comment:
It'd be great if we could simplify the helpers instead of adding `buttonLabel` argument everywhere.

Here's an idea:
- make the submit button in all the popovers actually use `type="submit"` 
- make helpers find the submit button using the `type`, not the label

WDYT?

Of course we can do this in a separate PR.

Reply:
Great idea. I can set it up in a follow up idea.